cli/azd/pkg/tools/swa/swa.go

```diff
@@ -44,16 +45,19 @@ type swaCli struct {
 	commandRunner exec.CommandRunner
 }
 
-func (cli *swaCli) Build(ctx context.Context, cwd string, appFolderPath string, outputRelativeFolderPath string) error {
-	_, err := cli.executeCommand(ctx,
-		cwd, "build",
-		"--app-location", appFolderPath,
-		"--output-location", outputRelativeFolderPath)
+func (cli *swaCli) Build(ctx context.Context, cwd string, buildProgress io.Writer) error {
+	fullAppFolderPath := filepath.Join(cwd)
+	result, err := cli.run(ctx, fullAppFolderPath, buildProgress, "build", "-V")
```

Comment:
yes, it is for verbose. This gives all the running logs if the swa cli fails. And there's no impact to azd because we use the showProgress component :)